Samples/StorageProviders/StorageProviders/BaseJSONStorageProvider.cs

```diff
@@ -15,6 +17,13 @@ namespace Samples.StorageProviders
     /// </summary>
     public abstract class BaseJSONStorageProvider : IStorageProvider
```

Comment:
With JSON I'm OK, but with Mongo I'm hesitant, as it would add this depdency into our core solution (albeit only in tests). Nevertheless maybe there's a way to use linked files at least for the base class and/or other providers? @sergeybykov any opinions with this?